.env.template

```
@@ -202,3 +202,6 @@ PRIVATE_TEST_REPO_NAME=<yourGithubUsername>/potpie-private-test-repo
 
 #stripe-potpie url for dodo payments
 STRIPE_POTPIE_URL=http://localhost:8003
+
+#Internal route between potpie and workflows.
+WORKFLOWS_MEDIA_INTERNAL_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)